src/Akka.Streams.Kafka/Helpers/CommitRefreshing.cs

```diff
@@ -46,24 +48,26 @@ public class NoOp<K, V> : ICommitRefreshing<K, V>
         public class Impl<K, V> : ICommitRefreshing<K, V>
         {
             private readonly TimeSpan _commitRefreshInterval;
-            private IImmutableSet<TopicPartitionOffset> _requestedOffsets = ImmutableHashSet<TopicPartitionOffset>.Empty;
-            private IImmutableSet<TopicPartitionOffset> _committedOffsets = ImmutableHashSet<TopicPartitionOffset>.Empty;
+            private IImmutableDictionary<TopicPartition, Offset> _requestedOffsets = ImmutableDictionary<TopicPartition, Offset>.Empty;
+            private IImmutableDictionary<TopicPartition, Offset> _committedOffsets = ImmutableDictionary<TopicPartition, Offset>.Empty;
```

Comment:
Yup, scala uses dictionaries in few places where I was using just sets with `TopicPartitionOffset` objects. Thanks for updating here, since looks like code becomes simpler in this case.